lib/parse.js

```diff
@@ -34,6 +34,8 @@ const route = (timezone, locations, products, remarks) => {
 	return (r) => {
 		const parsed = parse(r)
 		parsed.price = {amount: null, hint: 'No pricing information available.'}
+		parsed.platfar = r.arr.aPlatfS
```

Comment:
Would prefer a more descriptive name here, e.g. `departurePlatform`. Also, the documentation is missing.